### PR TITLE
Updating the swagger-spec files and adding a script to do that

### PR DESCRIPTION
**Pull request by nikhiljindal**
Files updated using the script.
We can use this script until we find a way to fix https://github.com/GoogleCloudPlatform/kubernetes/issues/4763

### CODE REVIEW

File: swagger-spec/update.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# https://rawgit.com/GoogleCloudPlatform/kubernetes/master/api/swagger-spec/
+# We convert the json files from there to index.html files here to act as a
+# server serving the spec for swagger-ui.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SOURCE_ROOT_PATH="https://rawgit.com/GoogleCloudPlatform/kubernetes/master/api/swagger-spec/"
+DESTINATION_ROOT_PATH=$(dirname "${BASH_SOURCE}")/..
```
Comment (vmarmol) [inline swagger-spec/update.sh]: Where is BASH_SOURCE defined?

Comment (burke) [inline swagger-spec/update.sh]: > ##### [`BASH_SOURCE`](https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html)
> An array variable whose members are the source filenames where the corresponding shell function names in the `FUNCNAME` array variable are defined. The shell function `${FUNCNAME[$i]}` is defined in the file `${BASH_SOURCE[$i]}` and called from `${BASH_SOURCE[$i+1]}`

Comment (vmarmol) [inline swagger-spec/update.sh]: Thanks @burke!

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**